internal/config/config.go

```diff
@@ -89,7 +89,8 @@ type Slack struct {
 
 // Artifacts represents the test artifacts configuration.
 type Artifacts struct {
-	Download ArtifactDownload `yaml:"download,omitempty" json:"download"`
+	Download        ArtifactDownload `yaml:"download,omitempty" json:"download"`
+	CleanupPrevious bool             `yaml:"cleanupPrevious,omitempty" json:"cleanupPrevious"`
```

Comment:
You don't think that `Cleanup` is concise yet intuitive enough? Especially since you aren't guaranteeing that you are cleaning up previously downloaded artifacts. You are deleting all contents of the artifacts folder.

Reply:
updated :)